### PR TITLE
Add brand pivot max-width

### PR DESCRIPTION
**Pull request by gustaveen**

### CODE REVIEW

File: src/components/blockHelpers.tsx [hunk 1]
```diff
@@ -37,6 +37,11 @@ export const CONTENT_MAX_WIDTH = {
   },
 }
 
+// New content max width
```
Comment (palmenhq) [inline src/components/blockHelpers.tsx]: do we need this comment? 😬

File: src/components/blockHelpers.tsx [hunk 2]
```diff
@@ -365,7 +373,11 @@ export const ContentWrapper: React.FC<ContentWrapperProps> = ({
     }}
   >
     {({ isVisible }) => (
-      <ContentWrapperStyled visible={index <= 1 || isVisible} {...props}>
+      <ContentWrapperStyled
+        visible={index <= 1 || isVisible}
+        brandPivot
```
Comment (palmenhq) [inline src/components/blockHelpers.tsx]: `brandPivot={brandPivot}` ?

**Review by palmenhq — COMMENTED** [review]
I feel like there's more blocks to change the `ContentWrapper` for but maybe not?

File: src/components/blockHelpers.tsx (hunk 2 above)
Comment (gustaveen) [reply src/components/blockHelpers.tsx]: Do we have a prefer to write the longer boolean syntax? :)

File: src/components/blockHelpers.tsx (hunk 1 above)
Comment (gustaveen) [reply src/components/blockHelpers.tsx]: Hehe nope, not necessary :) Just a reminder for us to remove the old max width later on.

File: src/components/blockHelpers.tsx (hunk 2 above)
Comment (gustaveen) [reply src/components/blockHelpers.tsx]: Yep, of course you need to do that when you pass down a prop, what was I thinking 😆

**Comment by gustaveen** [comment]
Did as we discussed @palmenhq and inverted the variable names 👍

**Review by palmenhq — APPROVED** [review]
wooh!

**Pull request merged**